packages/financial-templates-lib/test/price-feed/InsuredBridgePriceFeed.js

```diff
@@ -516,7 +516,7 @@ describe("InsuredBridgePriceFeed", function () {
 
       // Setting quote time after the latest block time will cause pricefeed to return 0.
       const depositTimestamp = Number(await bridgePool.methods.getCurrentTime().call());
-      const quoteTimestamp = Number((await web3.eth.getBlock("latest")).timestamp) + 60;
```

Comment:
This seems like we're making an opinionated change. Why remove the 60 second cushion?

Reply:
Yeah why remove?

Reply:
This test will pass as long as the quote time is > the relay block time. So yoh can grab the block time from the relay txn and just increment it. I think it's passing now by luck but it's not clear